### DISCUSSION TITLE
Project Page Navigation

### DISCUSSION THREAD

**Discussion by srothetafesa** [srothetafesa/Tafe24S1T109_Ext#4] [General]
Hi John,

I've been writing my code for the mortgage calculator, for the navigation between the main page and the calculator I used:

Frame.Navigate(typeof(MortgageCalculator));

I also used the same on the exit button in the calculator to go back to the main page. I figure we should use the same method to maintain consistency when we merge our files.

Let me know what you think.

Thanks,
Sam

**Comment by JohnLSison** [comment]
Hey Sam,
I will keep that in mind when coding the xaml code for my calculator's exit button as I think that is a good idea.
